crates/bevy_pbr/src/render/mesh_functions.wgsl

```diff
@@ -0,0 +1,34 @@
+fn mesh_model_position_to_world(vertex_position: vec4<f32>) -> vec4<f32> {
```

Comment:
I really like those changes, but maybe these functions should be named mesh_position_A_to_B and mesh_normal_A_to_B?

Reply:
I like that suggestion.